db/seeds.rb

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
+
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+after :playbooks do
```

Comment:
Might be worth putting a comment in this block explaining why it's empty in case someone thinks it's not doing anything and removes it later.

Reply:
I completely agree. I'll do that Sunday!